walletnode/services/artworkregister/task.go

```diff
@@ -128,17 +128,30 @@ func (task *Task) run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	defer finalImage.Remove()
+	log.WithContext(ctx).WithField("FileName", finalImage.Name()).Debugf("final image")
+	// defer finalImage.Remove()
 
 	if err := task.encodeFingerprint(ctx, fingerprint, finalImage); err != nil {
 		return err
```

Comment:
please wrap error with details where it failed : 
errors.Errorf("encode fingerprint failed:%w", err)